standard/types.md

```diff
@@ -681,10 +682,13 @@ The type `dynamic` uses dynamic binding, as described in detail in [§12.3.2](ex
 
 Because of this equivalence, the following holds:
 
-- There is an implicit identity conversion between `object` and `dynamic`, and between constructed types that are the same when replacing `dynamic` with `object`.
+- There is an implicit identity conversion
+  - between `object` and `dynamic`
+  - between constructed types that are the same when replacing `dynamic` with `object`
```

Comment:
I'll file an issue to say that this list shouldn't be needed here. We don't want to have to edit this section every time a new identity conversion between "things with identity convertible types" comes up.

Reply:
Raised #1034.